frontend/src/global_styles/content/_project_status.sass

```diff
@@ -1,11 +1,11 @@
 
-$project-status-gray: 204, 204, 204
-$project-status-red: #E73E3D
-$project-status-orange: #FFB030
-$project-status-green: #00D12D
-$project-status-deep-blue: #006E8F
-$project-status-light-blue: #84D6E5
-$project-status-deep-green: #007528
+$project-status-gray: var(--fgColor-subtle, var(--color-subtle-fg))
+$project-status-red: var(--fgColor-danger, var(--color-danger-fg))
+$project-status-orange: var(--fgColor-severe, var(--color-severe-fg))
+$project-status-green: var(--fgColor-success, var(--color-success-fg))
+$project-status-deep-blue: var(--fgColor-attention, var(--color-attention-fg))
+$project-status-light-blue: var(--fgColor-accent, var(--color-accent-fg))
+$project-status-deep-green: var(--fgColor-success, var(--color-success-fg))
```

Comment:
Please rename these variables. The variables don't necessarily translate to the colors in the names any more. Further, `status-green` and `status-deep-green` now have the same color. I know, that it was specified like this, so I asked the product team in the ticket to double check.